codec-http2/src/main/java/io/netty/handler/codec/http2/StreamWriteDistributor.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.http2;
+
+/**
+ * An object (used by remote flow control) that is responsible for distributing the bytes to be
+ * written across the streams in the connection.
+ */
+public interface StreamWriteDistributor {
```

Comment:
I don't think so ... it's not used by the `Http2RemoteFlowController` interface.  I think it's best as a stand-alone interface for now.